index.html

```diff
@@ -228,8 +228,28 @@ <h3 class="about-list"><span>Skills</span> <i class="fas fa-chevron-right"></i><
         </ul>
     </section>
 
-   
-   </main>   
+  
+  <footer class="contact-section">
+     <div class="contact-container">
+            <h2>Contact me</h2>
+          <div> 
+             <p>If you have an application you are interested in developing,
+              a feature that you need built or a project that needs coding.
+              I’d love to help with it</p> 
+            </div>
+        </div>
+        
+        <form class="input-section">
+           <ul>
+           <li><input type="text" placeholder="your name" maxlength="30" required></li>
+           <li><input type="email" placeholder="Enter your email" required></li>
+           <li><textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea></li>
+          </ul>
+          <button class="btn" type="submit">Get It Touch</button>
+        </form>      
```

Comment:
- [ ] How about removing the form inputs and textarea from a `<ul>`? I am suggesting this because we are currently on dealing with a list. So it would be nice if you maintained a very simple structure for the form just to improve the overall code readability as seen below.

```
<form class="input-section">
  <input type="text" placeholder="your name" maxlength="30" required>
  <input type="email" placeholder="Enter your email" required>
  <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
  <button class="btn" type="submit">Get It Touch</button>
</form>
```

- [ ] The other suggestion is that it would be nice if you aligned the form children using flexbox because it aligns items in a clean simple way with less code 😄.